multidim-interop/impl/js/v0.46/Dockerfile

```
@@ -1,11 +1,9 @@
 # Here because we want to fetch the node_modules within docker so that it's
 # installed on the same platform the test is run. Otherwise tools like `esbuild` will fail to run
-FROM node:18
+FROM node:18.17.1
 WORKDIR /app
-COPY . .
-RUN npm i && npm run build
+RUN npm install @libp2p/multidim-interop@1.0.0-c4eff4c5
```

```suggestion
RUN apt-get update && apt-get install -y cmake
RUN npm install @libp2p/multidim-interop@1.0.0-c4eff4c5
```

This comment needs to be addressed: https://github.com/libp2p/js-libp2p/pull/2008#issuecomment-1703067528
When I try to build this locally without that patch this fails at the cmake step

---

Good catch thanks!